biorxiv/corpora_comparison/word_frequency_counts.ipynb

```diff
@@ -12,39 +12,35 @@
    "execution_count": 1,
    "metadata": {
     "ExecuteTime": {
-     "end_time": "2020-05-16T15:35:08.603985Z",
-     "start_time": "2020-05-16T15:35:08.005652Z"
+     "end_time": "2020-08-07T15:31:55.121146Z",
+     "start_time": "2020-08-07T15:31:54.460344Z"
     }
    },
    "outputs": [],
    "source": [
     "%load_ext autoreload\n",
     "%autoreload 2\n",
     "\n",
-    "from collections import Counter, defaultdict\n",
+    "from collections import Counter\n",
     "import csv\n",
     "from pathlib import Path\n",
     "import pickle\n",
     "import string\n",
-    "import sys\n",
     "\n",
-    "sys.path.append(\"../../modules/\")\n",
-    "\n",
-    "import gensim\n",
     "import pandas as pd\n",
     "import spacy\n",
-    "from tqdm import tqdm_notebook\n",
+    "from tqdm import tqdm\n",
     "\n",
-    "from document_helper import dump_article_text"
+    "from annorxiver_modules.document_helper import dump_article_text"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 2,
    "metadata": {
     "ExecuteTime": {
-     "end_time": "2020-05-16T15:35:09.118438Z",
-     "start_time": "2020-05-16T15:35:08.605318Z"
+     "end_time": "2020-08-07T15:31:55.658944Z",
+     "start_time": "2020-08-07T15:31:55.122472Z"
```

Comment:
>Do you need the extra doi column since that information is in the index? Perhaps you're using it some other place though

Good point. It is redundant information as agg provides the doi in the index column; however, I usually ignore the index when I'm writing my code. This results in the index being reset eventually. 

>What does disable = ['ner'] do?

So Spacy's parser provides Named Entity Recognition as part of its parser. Since I only care about the part of speech tags and dependency tags, I disabled that feature to speed up computation.

> Would it make sense to have cell 5 be made into a function since you're using it twice (for bioRiv and PMC)? 

Good point. I'll reformat to do that.

> Also your cells is labeled as word frequency which confused me a little because you're returning sentence lengths. But perhaps I'm misunderstanding something here. My assumption is that word frequency refers to the number of times the word dog appears versus sentence length is the number of words in a given sentence. Is that right?

So I'm combining this function to be return word frequency, part of speech tag freq, dependency tag freq, as well as obtaining the length of every sentence within each document. This sentence length is for my comparison table that is needed for my manuscript. Basically I'm trying to do all the things in one go.